app-backend/api/src/main/scala/user/Routes.scala

```diff
@@ -271,4 +280,61 @@ trait UserRoutes
       }
     }
   }
+
+  def createUserBulk: Route = authenticate { user =>
+    authorizeScope(
+      ScopedAction(Domain.Organizations, Action.AddUser, None),
```

Comment:
Should anyone with `addUser` powers be able to do this? I know that's already pretty limited so probably it's fine, I'm just curious whether this is a deliberate power granted to all future org / platform admins

Reply:
I think this is a good point - offline @aaronxsu and I also discussed something related to this.

I think this should be a separate scope since we only need this endpoint for a single application and I think it's good to be a little defensive here and not accidentally give people too much power.

Reply:
Agreed. I will add a new scope.